python/semantic_kernel/orchestration/sk_function_base.py

```diff
@@ -199,3 +199,10 @@ def set_ai_configuration(
             SKFunctionBase -- The function instance
         """
         pass
+
+    def __repr__(self):
+        """
+        Override how the classes are printed as strings so we don't include
+        the class attributes.
+        """
+        return f"{self.__class__.__name__}()"
```

Comment:
Does it need to have the `()`? Thinking that it could mislead to think is a class/function with no arguments.

Reply:
Thanks for your feedback. I agree that including () would be misleading, as it suggests that instances of SKFunctionBase could be created without any arguments, which isn't the case here (given that it's an ABC).